rust/operator-binary/src/config.rs

```diff
@@ -36,6 +36,9 @@ pub const PYTHON_IMPORTS: &[&str] = &[
     "from log_config import StackableLoggingConfigurator",
     ];
 
+pub const OPA_IMPORTS: &[&str] =
+    &["from opa_authorizer.opa_manager import OpaSupersetSecurityManager"];
```

Comment:
Can you please move that constant into `rust/operator-binary/src/authorization/opa.rs`?

Reply:
[done](https://github.com/stackabletech/superset-operator/pull/582/commits/0d078b1f4c8932fee75ced030db6b50ed75c39a4)